pkg/network/multus_dynamic_networks_controller.go

```diff
@@ -32,7 +32,7 @@ func renderMultusDynamicNetworks(conf *cnao.NetworkAddonsConfigSpec, manifestDir
 	} else {
 		data.Data["CniMountPath"] = cni.BinDir
 	}
-	data.Data["Placement"] = conf.PlacementConfiguration.Workloads
+	data.Data["Placement"] = conf.PlacementConfiguration.Infra
```

Comment:
multus-dynamic-networks-controller deploys a DaemonSet. So it should be Workloads, no?

Reply:
Make sense, also the passt installer is a daemonset, fixing that too.